modules/ebi-metagenomics/eggnogmapper/main.nf

```diff
@@ -9,38 +9,48 @@ process EGGNOGMAPPER {
 
     input:
     tuple val(meta), path(fasta)
-    path(eggnog_db)
+    tuple val(meta2), path(annotation_hit_table)
     path(eggnog_data_dir)
+    path(eggnog_db)
     path(eggnog_diamond_db)
 
     output:
-    tuple val(meta), path("*.emapper.hits"), emit: csv
-    path "versions.yml",                     emit: versions
+    tuple val(meta), path("*.hits")   , emit: hits, optional: true
```

Comment:
don't forget the extra empty spaces behind the comma now :D

Reply:
hahahahahaha there will always be at least _one_ of these I forget it seems 😄 thanks for the review!